src/client/app/components/dashboard/dashboard.component.ts

```diff
@@ -10,7 +10,7 @@ import {HeroService} from '../../services/hero.service';
     styleUrls: ['app/components/dashboard/dashboard.component.css']
 })
 
-export class DashboardComponent  implements OnInit {
+export class DashboardComponent implements OnInit {
     heroes: Hero[] = [];
 
     constructor(
```

Comment:
Try not to commit something you didn't really modify. Clouds up the amount of code in the PR.